docs/compute/drivers/auroracompute.rst

```diff
@@ -1,5 +1,5 @@
 AuroraCompute Computer Driver Documentation
-======================================
+===========================================
```

Comment:
We need to update our build process to turn sphinx warnings into errors again and fail the build.

It used to be turned on, but then for some reason (don't recall why anymore), we disabled it.